.github/workflows/build.yml

```diff
@@ -7,6 +7,7 @@ on:
     - master
     - main
     - feature-**
+    - rquitales/fix-disk-space-ci
```

Comment:
To test that this solution works before merging to `master`. Will be removed on submit.